### DISCUSSION TITLE
在英文状态怎么显示单词推荐呢

### DISCUSSION THREAD

**Discussion by pzb568** [fcitx5-android/fcitx5-android#390] [Q&A | 问答] [answered]
英文状态下希望可以加入单词推荐，我在设置找了，但是没有发现这个功能，对于英文不会的人非常有用，希望可以加入这个功能，非常感谢

**Comment by rocka** [comment, marked as answer]
在英文输入状态下，点击键盘左上角箭头展开工具栏，再点击工具栏展开后最右边的 <kbd>···</kbd> 按钮打开 StatusArea ，启用“单词提示”

或者

输入法 - English - 启用单词提示

或者

附加组件 - Android 英文键盘 - 启用单词提示

**Reply by pzb568** [reply]
好的，谢谢
